SU-HSDO/suhumsci
# fix(shs-5063): adjustments to link/icon color in dark mode

# READY FOR REVIEW

## Summary
- Updates PDF, Lock, and File icons in local footer in Colorful theme in dark mode in all color schemes to white / gray medium on hover
- Updates links on Vertical Linked Cards to be white with gray medium hover for Colorful theme
- Updates links on Vertical Linked Card to be black with gray dark hover for Traditional theme

## Urgency
medium

## Steps to Test
1. View the site in Colorful theme. 
2. Ensure the local footer is set to dark mode
3. In all color schemes, make sure the lock, pdf and file icons show in the local footer as white with gray hover. 
4. Visit a page with Vertical Links Cards and add a link to one, like here: http://hs-colorful.suhumsci.loc/components/collections-cards
5. Ensure that links are white with gray hover*
6. Switch to view site in Traditional theme. 
7. Visit a page with Vertical Links Cards and add a link to one, like here: http://hs-traditional.suhumsci.loc/components/collections-cards
8. Ensure that links are black with dark gray hover*

* There is a major issue where the links are just not clickable in this kind of card. I'm not sure it should be an option here. There is no way to click since the whole card is covered in an absolutely positioned link.
